static/src/stylesheets/module/football/_overview.scss

```diff
@@ -1,3 +1,160 @@
+/* ==========================================================================
+   World Cup 2018
+   ========================================================================== */
+.wc-overview {
+    .container__title {
+        &:after {
+            display: none;
+        }
+    }
+}
+
+.facia-header {
+    .facia-header__inner {
+        padding-top: $gs-baseline / 2;
+        background-color: $news-garnett-highlight;
+        padding-bottom: 0px;
+        display: flex;
+        flex-direction: column;
+
+        @include mq($from: desktop) {
+            height: gs-span(1);
+            flex-direction: row;
+        }
+    }
+
+    .facia-header__column {
+        flex-basis: 100%;
+        overflow: visible;
+        border-bottom: 1px solid #121212;
+        padding-bottom: $gs-gutter / 2;
+
+        @include mq($from: desktop) {
+            border-bottom: 0;
+            padding-bottom: 0;
+            flex-basis: 380px;
```

Comment:
Would it be possible to use the `gs-span()` function to define widths, and try to utilise our grid system?